Lecture-4/Task4/Task4/Runner.java

```diff
@@ -0,0 +1,29 @@
+package Task4;
+
+public class Runner {
+
+    private String name;
+    private int time;
+
+    public void setName(String name) {
+        this.name = name;
+    }
+    public String getName(){
+        return name;
+    }
+    public void setTime(int time) {
+        this.time = time;
+    }
+    public int getTime() {
+        return time;
+    }
+
+    public Runner(String name,int time){
+        this.name = name;
+        this.time = time;
+    }
+
+    public void showRunner(){
```

Comment:
Better to override `toString()` method instead of creating new method with similar goal

Reply:
done